includes/definitions.inc.php

```diff
@@ -1435,6 +1435,19 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+// Lenovo EMC (NAS)
+$os = 'lenovoemc';
```

Comment:
Yep, there is a web interface - but I admit, they don't call it much (and it isn't much .. LOL). It just says "lenovo" at the top, and then config options. Status page -> just Lenovo, then HW and SW status / details.

I'm definitely open to suggestions!